k8s/nginx/apptest/tester/tests/basic-suite.yaml

```diff
@@ -1,8 +1,36 @@
 actions:
-- name: dummy test
+- name: Wait until NGINX is ready
   bashTest:
     script: |-
-      exit 0
+      declare -i replicas=$(kubectl get sts "${APP_INSTANCE_NAME}-nginx" \
+        --namespace "${NAMESPACE}" \
+        --output jsonpath='{.spec.replicas}')
+
+      for ((pod=0; pod<${replicas}; pod++)); do
+        kubectl wait pod/${APP_INSTANCE_NAME}-nginx-${pod} \
+          --namespace=${NAMESPACE} \
+          --for=condition=Ready \
+          --timeout=180s
+      done
     expect:
       exitCode:
         equals: 0
+- name: Site address should be 200 OK
+  httpTest:
+    url: http://${EXTERNAL_IP}
```

Comment:
Actually, this is reasonable FR for Testrunner, to support environment variables in httpTest. Could you add TODO here and issue in testrunner?

Reply:
This is done - https://github.com/GoogleCloudPlatform/marketplace-testrunner/issues/5